### ISSUE TITLE
Change nodejs reference generation

### ISSUE THREAD

**Issue by Dr-Electron** [iotaledger/iota-sdk#751]
## Description

Every subsection page name starts with its type. We can get rid of that.
Instead of `Class: Account` it should just be `Account`

## Are you planning to do it yourself in a pull request?

Yes.

**Comment by Dr-Electron** [comment]
I'm actually not sure how to do this. Seems to be "hardcoded". We could add another frontmatter plugin, but not sure if those can be dynamic. I opened an issue in their repo to ask what the best solution would be. If anyone has any other ideas, tell me

**Comment by thibault-martinez** [comment]
Any news on this?

**Comment by Dr-Electron** [comment]
Not really. We would need to solve this: https://github.com/tgreyuk/typedoc-plugin-markdown/issues/450

**Comment by thibault-martinez** [comment]
Alright, I'll move it to a later release then, so I don't annoy you for a couple of weeks.

**Comment by Dr-Electron** [comment]
Blocked until we successfully upgraded to v3 of docusaurus
